packages/babel-generator/test/fixtures/flow/declare-exports/expected.js

```diff
@@ -38,4 +38,4 @@ declare export opaque type Bar<T>;
 declare export opaque type Baz: Foo;
 declare export opaque type Foo<T>: Bar<T>;
 declare export opaque type Foo<T>: Bar;
-declare export opaque type Foo: Bar<T>;
+declare export opaque type Foo: Bar<T>;
```

Comment:
I hate that this is adding files with no line terminators, but it seems to be a problem with how `babel-generator` is writing flow nodes.